desktop_server/src/com/mygdx/game/server/model/Client.java

```diff
@@ -7,4 +7,6 @@
 public class Client {
   private int id;
   private final ServerWebSocket socket;
+  private String playerUsername;
+  private String playerToken;
```

Comment:
O, to Client zamienia się w gracza

Reply:
A w sumie w serwerze trochę tak, :P co o tym myślisz?